lib/api/src/grpc/proto/collections.proto

```diff
@@ -148,15 +148,13 @@ message OptimizersConfigDiff {
 
 message CreateCollection {
   string collection_name = 1; // Name of the collection
-  optional uint64 vector_size = 2 [deprecated = true]; // Size of the vectors
-  optional Distance distance = 3 [deprecated = true]; // Distance function used for comparing vectors
-  optional HnswConfigDiff hnsw_config = 4; // Configuration of vector index
-  optional WalConfigDiff wal_config = 5; // Configuration of the Write-Ahead-Log
-  optional OptimizersConfigDiff optimizers_config = 6; // Configuration of the optimizers
-  optional uint32 shard_number = 7; // Number of shards in the collection, default = 1
-  optional bool on_disk_payload = 8; // If true - point's payload will not be stored in memory
-  optional uint64 timeout = 9; // Wait timeout for operation commit in seconds, if not specified - default value will be supplied
-  optional VectorsConfig vectors_config = 10; // Configuration for vectors
+  optional HnswConfigDiff hnsw_config = 2; // Configuration of vector index
```

Comment:
We have to keep field indices because of backward compatibility. Try to reserve indices for removed fields:
https://developers.google.com/protocol-buffers/docs/proto3#reserved